plaso/multi_processing/psort.py

```diff
@@ -742,7 +746,7 @@ def _UpdateProcessingStatus(self, pid, process_status, used_memory):
 
     process = self._processes_per_pid[pid]
 
-    processing_status = process_status.get('processing_status', None)
+    status_indicator = process_status.get('processing_status', None)
```

Comment:
I think the renaming here is good, given the large number of "status" things (ProcessStatus vs ProcessingStatus), but maybe the definitions need to be renamed to, to STATUS_INDICATOR_IDLE etc. 
These definitions are sometimes called "status" and sometimes called "status_indicator" which is a bit confusing.

Reply:
Moved AI to separate issue https://github.com/log2timeline/plaso/issues/2437